src/Containers/packaging/build/Microsoft.NET.Build.Containers.targets

```diff
@@ -99,10 +99,10 @@
 
     <ItemGroup Label="AppCommand Assignment" Condition="'$(ContainerAppCommandInstruction)' != 'None'">
       <!-- For self-contained, invoke the native executable as a single arg -->
-      <ContainerAppCommand Condition="'$(ContainerAppCommand)' == '' and '$(_ContainerIsSelfContained)' == 'true' and !$(_ContainerIsTargetingWindows)" Include="$(ContainerWorkingDirectory)/$(AssemblyName)$(_NativeExecutableExtension)" />
-      <ContainerAppCommand Condition="'$(ContainerAppCommand)' == '' and '$(_ContainerIsSelfContained)' == 'true' and $(_ContainerIsTargetingWindows)" Include="$(AssemblyName)$(_NativeExecutableExtension)" />
+      <ContainerAppCommand Condition="'$(ContainerAppCommand)' == '' and $(_ContainerIsSelfContained)" Include="$(ContainerWorkingDirectory)/$(AssemblyName)$(_NativeExecutableExtension)" />
```

Comment:
Were you able to consolidate these because `_NativeExecutableExtension` is empty on non-Windows platforms?

Reply:
Correct :)

Reply:
I first started off by the tests asserting that the executable on Windows should be `C:\apps\project.exe`, but I see that the path separators seem to be consolidated to `/` on both win and *nix platforms, so I guess that is intended (I know Windows has supported `/` as a path separator for decades, but it doesn't seem to catch on with the public :) )